cnxarchive/views.py

```diff
@@ -567,6 +698,10 @@ def search(request):
         page = None
     if page is None or page <= 0:
         page = 1
+    try:
+        in_book_uuid = params.get('uuid', '')
+    except (TypeError, ValueError, IndexError):
+        in_book_uuid = None
```

Comment:
is this leftover from trying to overload the search method? Not sure what it's doing.